tests/test_bs.py

```diff
@@ -325,20 +327,20 @@ def save_log_on_exit(request):
 current_index = 'kubernetes_cluster-' + todaydate
 
 
-def test_bootstrap(setup_bootstrap):
+def test_bootstrap(set_namespace, setup_bootstrap):
```

Comment:
there isn't a more elegant way of using `set_namespace` instead of prepending it to every func ? like in a higher scope or something.

Reply:
It is in a higher scope. The scope is "session" see fixtures.py

Reply:
I meant in a way that won't require writing it in every test. (because as it seems it's a requirement for every test.) when running a suite we want to use the same namespace for each test or use a different one for each test?